AssetLoader/interface/GLTFLoader.hpp

```diff
@@ -410,19 +410,22 @@ struct Primitive
     const Uint32 FirstIndex;
     const Uint32 IndexCount;
     const Uint32 VertexCount;
+    const Uint32 VertexStart;
```

Comment:
I suggest naming the variable `FirstVertex` to make it consistent with `FirstIndex`.
Also, add it before the `VertexCount` to make the ordering consistent as well.